rest-api-spec/src/main/resources/rest-api-spec/test/indices.data_stream/20_unsupported_apis.yml

```diff
@@ -31,11 +31,6 @@
       indices.delete:
         index: logs-foobar
 
-  - do:
```

Comment:
I think we should add a test to verify that doing `indices.get` on a data stream gives the right output (at least set of indices). Same for mappings.